tests/tool-bazel.yaml

```
@@ -0,0 +1,5 @@
+- desc: bazel cache should be persisted to /workspaces
+  command: [file, /workspace/.bazel]
+  assert:
+  - status == 0
+  - stdout.indexOf("/workspace/.bazel: symbolic link to /home/gitpod/.cache/bazel") != -1
```

I think it should be something like this: [🔨 `Dazzle build`](https://github.com/gitpod-io/workspace-images/runs/7421114218?check_suite_focus=true#step:10:5) step is failing

```suggestion
  - stdout.indexOf("symbolic link to /home/gitpod/.cache/bazel") != -1
```

---

OR

```suggestion
  - stdout.indexOf("/workspace/.bazel") != -1
```